package-lock.json

```diff
@@ -1,8 +1,8481 @@
 {
   "name": "azure-rest-api-specs-tests",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "azure-rest-api-specs-tests",
+      "version": "0.1.0",
```

Comment:
The change in this file is auto generated. correct?

Reply:
Will remove it